tutorials/Titanic_Basic_Interpret.ipynb

```diff
@@ -835,7 +835,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.7.6"
```

Comment:
Looks like python version changed. Here. We should probably agree on one and use that for all notebooks.

Reply:
I think the notebook should run the same for every Python version, I just had a different one than @vivekmig 's.